go-controller/pkg/kubevirt/switch.go

```diff
@@ -0,0 +1,22 @@
+package kubevirt
```

Comment:
should this have a unit test ?

Reply:
It will already tested at kubevirt_test with a more integrated fashion, I think is overkill to unit test a so simple and declarative function.